src/negotiation/taker/order.ts

```diff
@@ -23,21 +23,23 @@ export interface MatchingCriteriaAsset {
   maxNominalAmount?: string;
 }
 
+export { Order as TakerOrder };
+
 /**
  * Handles an order for the taker. It has helper functions to facilitate the handler of an
- * order by a taker. This should only be instantiated via `TakerNegotiatior.getOrder()` and should not be constructed from
+ * order by a taker. This should only be instantiated via {@link getOrder} and should not be constructed from
  * scratch.
  * @param rawParams - The raw parameters of the order.
  * @param criteria - The criteria used to get this order.
- * @param takeOrder - Function passed from the `Negotiator` to coordinate calls to `cnd` and the maker to effectively
+ * @param takeOrder - Function passed from the {@link TakerNegotiator} to coordinate calls to `cnd` and the maker to effectively
  * take the order and start the atomic swap execution.
  */
-export class Order {
+class Order {
```

Comment:
Am i seeing double? :D 
Should this description be on top of the constructor below?

Reply:
removed the parameters from the class, amended the parameter dscr on the constructor.